shared/enums/ticket-type.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum TicketType {
+  ADULT = "Adult",
+  STUDENT = "Student",
+  PENSIONER = "Pensioner",
```

Comment:
The same goes here, I think we should use SENIOR instead of PENSIONER

Reply:
Changed name to senior.